src/nccl.h.in

```diff
@@ -31,7 +31,9 @@ typedef struct ncclComm* ncclComm_t;
 #define NCCL_COMM_NULL NULL
 
 #define NCCL_UNIQUE_ID_BYTES 128
-typedef struct { char internal[NCCL_UNIQUE_ID_BYTES]; } ncclUniqueId;
+typedef struct {
+    char internal[NCCL_UNIQUE_ID_BYTES];
+} ncclUniqueId __attribute__((aligned(8)));
```

Comment:
This leads to align-mismatch at the clang: https://godbolt.org/z/zE5zdca5b

__attribute__((aligned(8))) property should be placed before the type name:
typedef struct {
    char internal[NCCL_UNIQUE_ID_BYTES];
}  __attribute__((aligned(8))) ncclUniqueId;

https://godbolt.org/z/reh9cbEvx